superx/tests/test_remove_item.py

```diff
@@ -0,0 +1,24 @@
+'''
+import client session
+'''
+import pytest
+from flask import session
+
+
+@pytest.mark.run(order=5)
+def test_remove_item(client):
+    '''
+    tests removing item from comparing cart
+    '''
+    test_item = {
+        'id': '45232123',
+        'name': 'mushrooms',
+        'quantity': '2',
+        'is_weighted': 'Flase',
+        'unit_of_measure': 'גרם'
+    }
+
+    with client:
+        client.post('/addItem', data=test_item, follow_redirects=True)
+        client.post('/removeItem', data=test_item, follow_redirects=True)
+        assert session['cart'] == []
```

Comment:
This assumes that the cart was empty before you started.
If test_add_item.py is executed before this test, there will be an item in the cart before you start.
It may make more sense to first save what is in the cart:
        oldcart = session['cart'].copy() # use copy instead of just = so you aren't saving a reference to the cart
        client.post('/addItem', data=test_item, follow_redirects=True)
        client.post('/removeItem', data=test_item, follow_redirects=True)
        assert session['cart'] == oldcart

Reply:
that sounds right,
I added oldcart to the code, then removed the item added in test_item and confirmed session is empty

Reply:
after testing the changes, I realize that every test creates an independent session.
that's why in remove_item.py session I cannot find the items added from add_item.py

*changed back to origin*